packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -204,482 +154,281 @@ void main() {
       });
 
       test('finds by ValueKey', () async {
-        when(mockIsolate.invokeExtension(any, any)).thenAnswer((Invocation i) {
-          expect(i.positionalArguments[1], <String, String>{
-            'command': 'tap',
-            'timeout': _kSerializedTestTimeout,
-            'finderType': 'ByValueKey',
-            'keyValueString': 'foo',
-            'keyValueType': 'String',
-          });
-          return makeMockResponse(<String, dynamic>{});
-        });
         await driver.tap(find.byValueKey('foo'), timeout: _kTestTimeout);
+        expect(fakeClient.commandLog, <String>[
+          'ext.flutter.driver {command: tap, timeout: $_kSerializedTestTimeout, finderType: ByValueKey, keyValueString: foo, keyValueType: String}',
+        ]);
       });
     });
 
     group('BySemanticsLabel', () {
       test('finds by Semantic label using String', () async {
-        when(mockIsolate.invokeExtension(any, any)).thenAnswer((Invocation i) {
-          expect(i.positionalArguments[1], <String, String>{
-            'command': 'tap',
-            'timeout': _kSerializedTestTimeout,
-            'finderType': 'BySemanticsLabel',
-            'label': 'foo',
-          });
-          return makeMockResponse(<String, dynamic>{});
-        });
         await driver.tap(find.bySemanticsLabel('foo'), timeout: _kTestTimeout);
+        expect(fakeClient.commandLog, <String>[
+          'ext.flutter.driver {command: tap, timeout: $_kSerializedTestTimeout, finderType: BySemanticsLabel, label: foo}',
+        ]);
       });
 
       test('finds by Semantic label using RegExp', () async {
-        when(mockIsolate.invokeExtension(any, any)).thenAnswer((Invocation i) {
-          expect(i.positionalArguments[1], <String, String>{
-            'command': 'tap',
-            'timeout': _kSerializedTestTimeout,
-            'finderType': 'BySemanticsLabel',
-            'label': '^foo',
-            'isRegExp': 'true',
-          });
-          return makeMockResponse(<String, dynamic>{});
-        });
         await driver.tap(find.bySemanticsLabel(RegExp('^foo')), timeout: _kTestTimeout);
+        expect(fakeClient.commandLog, <String>[
+          'ext.flutter.driver {command: tap, timeout: $_kSerializedTestTimeout, finderType: BySemanticsLabel, label: ^foo, isRegExp: true}',
+        ]);
       });
     });
 
     group('tap', () {
       test('requires a target reference', () async {
-        expect(driver.tap(null), throwsDriverError);
+        expect(driver.tap(null), throwsAssertionError);
```

Comment:
Several of these changed and I'm not entirely sure why.

I'm not super concerned about it though, since before long we'll be migrating this to NNBD and this test will just go away anyway.

Reply:
(The test and the entire class of error will change)